src/components/Carousel/CarouselCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const CarouselCard = ({ video }) => {
+  return (
+    <article className="carousel-card">
+      <img src={video.thumbnail} alt={video.title} />
+      <button
+        className="btn btn-solid-secondary btn-rc"
+        onClick={() => console.log("clicked")}
+      >
```

Comment:
Remove console.log

Reply:
Have't not implemented the functionality part yet, will remove it and replace it with other method